src/features/DisplayControls/NursingTasks/components/NursingTasks.jsx

```diff
@@ -186,6 +199,21 @@ export default function NursingTasks(props) {
         ) : (
           <div className="nursing-task-tiles-container">{showTaskTiles()}</div>
         )}
+        {showSuccessNotification && (
+          <Notification
+            hostData={{
+              notificationKind: "success",
+              messageId: "NURSING_TASKS_SAVE_MESSAGE",
+            }}
+            hostApi={{
+              onClose: () => {
+                setShowSuccessNotification(false);
+                refreshDisplayControl([componentKeys.NURSING_TASKS]);
+                refreshDisplayControl([componentKeys.DRUG_CHART]);
```

Comment:
refresh display control supports multiple array in case we need to update multiple components so pass both keys as part of array and not calling refreshDisplayControl twice

Reply:
removed drug chart component for now as this will be taken care as a part of different story